app/lib/redirect.mjs

```diff
@@ -0,0 +1,35 @@
+import data from '@begin/data'
+
+const redirects = {
+  '/backend-workshop': '/workshop',
+  '/learn-backend': '/workshop',
+  '/backend': '/workshop',
+}
+
+const snakeToCamel = (str) =>
+  str
+    .toLowerCase()
+    .replace(/([-_][a-z])/g, (group) =>
+      group.toUpperCase().replace('-', '').replace('_', '')
+    )
+
+export async function checkRedirects(req) {
+  const path = req.requestContext.http.path
+  const isPath = Object.keys(redirects).includes(path)
```

Comment:
Extremely minor: maybe `isRedirectPath` for clarity?

Reply:
Whenever I see a method that starts with `is` I think it will return a boolean. In this case, the method will redirect the user if the path requested is in the redirects object.